tests/testTriggers.go

```diff
@@ -2095,6 +2095,8 @@ func TriggerStorageFullPoolExpansion(contexts *[]*scheduler.Context, recordChan
 						if newPoolSize >= expectedSizeWithJournal {
 							isExpansionSuccess = true
 						}
+						log.InfoD("Printing The storage pool status after pool expansion")
```

Comment:
How about including node details like Name or Id in the log message?

Reply:
Added the name for all the log statements.